src/api/TencentMeeting.ts

```diff
@@ -271,5 +271,41 @@ export async function getFileAddresses(recordFileId: string, tmUserId: string) {
   return res;
 }
 
+// https://cloud.tencent.com/document/product/1095/105659
+export async function getSmartSpeakers(recordFileId: string, tmUserId: string) {
```

Comment:
specify return type

Reply:
The return type would be a promise and none of the other APIs in `TecentMeeting.ts` specified a return type. A shared type is specified in `shared/summary.ts` and used to ensure the types match each other.